### PR TITLE
3.77.0 changelogs

### PR DESCRIPTION
**Pull request by c2thorn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
 ## 3.77.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_scc_notification_config` ([#9578](https://github.com/hashicorp/terraform-provider-google/pull/9578))
+
+IMPROVEMENTS:
+* composer: added field `maintenance_window` to resource `google_composer_environment` (beta) ([#9595](https://github.com/hashicorp/terraform-provider-google/pull/9595))
```
Comment (melinath) [inline CHANGELOG.md:7]: This is beta-only

**Review by melinath — CHANGES_REQUESTED** [review]
LGTM apart from the one thing

**Review by melinath — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+
+FEATURES:
+* **New Resource:** `google_scc_notification_config` ([#9578](https://github.com/hashicorp/terraform-provider-google/pull/9578))
+
+IMPROVEMENTS:
+* compute: fixed a permadiff bug in `log_config` field of `google_compute_region_backend_service` ([#9568](https://github.com/hashicorp/terraform-provider-google/pull/9568))
+* dlp: added `crypto_replace_ffx_fpe_config` and `crypto_replace_ffx_fpe_config` as primitive transformation types to `google_data_loss_prevention_deidentify_template` ([#9572](https://github.com/hashicorp/terraform-provider-google/pull/9572))
+
+BUG FIXES:
+* Fixed ordering of `budget_filter. projects` on `google_billing_budget` ([#9598](https://github.com/hashicorp/terraform-provider-google/pull/9598))
```
Comment (megan07) [inline CHANGELOG.md:11]: Does this need a service in front of it? billing maybe?

**Review by megan07 — APPROVED** [review]

**Pull request merged**